ksql-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedDeserializer.java

```diff
@@ -35,14 +35,27 @@
 public class KsqlDelimitedDeserializer implements Deserializer<GenericRow> {
 
   private final Schema schema;
+  private final CSVFormat csvFormat;
   private final StructuredLogger recordLogger;
   private final ProcessingLogContext processingLogContext;
 
+  public KsqlDelimitedDeserializer(final Schema schema) {
```

Comment:
Can you drop these two new constructors.  They are only used in your tests. In their place just use mock `StructuredLogger` and `ProcessingLogContext` in your tests using mockito.

(This will also fix your compiler error)